### PR TITLE
[RFXCOM] Make HW/FW versions available

### PR DESCRIPTION
**Pull request by mjagdis**
Hardware and firmware versions shouldn't need too much effort to find. Let's log them on connect and put them in the bridge thing's properties for the GUIs.

Signed-off-by: Mike Jagdis <mjagdis@eris-associates.co.uk> (github: mjagdis)

### CODE REVIEW

File: addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComBridgeHandler.java [hunk 1]
```diff
@@ -232,10 +236,12 @@ public void packetReceived(byte[] packet) {
                 if (message instanceof RFXComInterfaceMessage) {
                     RFXComInterfaceMessage msg = (RFXComInterfaceMessage) message;
                     if (msg.subType == SubType.RESPONSE) {
-                        logger.debug("RFXCOM transceiver/receiver type: {}, hw version: {}.{}, fw version: {}",
+                        if (msg.command == Commands.GET_STATUS) {
+                            logger.info("RFXCOM transceiver/receiver type: {}, hw version: {}.{}, fw version: {}",
                                 msg.transceiverType, msg.hardwareVersion1, msg.hardwareVersion2, msg.firmwareVersion);
+                            getThing().setProperty(PROPERTY_HARDWARE_VERSION, msg.hardwareVersion1 + "." + msg.hardwareVersion2);
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComBridgeHandler.java]: Could you use the default properties for this https://github.com/eclipse/smarthome/blob/master/bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/Thing.java#L50

**Comment by martinvw** [comment]
@mjagdis I took the liberty to change the reference to the static field, its best practice not to access static fields via an instance, although you do that... See also https://stackoverflow.com/questions/3630485/whats-the-best-way-to-fix-this-write-to-static-field-from-instance-method-fin

**Comment by martinvw** [comment]
EDIT: I missed the word 'can'

**Comment by martinvw** [comment]
@mjagdis Do you get proper output for this?

> Current firmware version: 12
> hardwareVersion  0.1

I have a vague memory that we did port the version number support from old openHAB binding, but this looks not correct.

**Comment by mjagdis** [comment]
It's correct as far as the version of the SDK I have is concerned. The version numbers quoted for rfxcom downloads leave me wondering if the hw numbers should be the other way round and prepended to the fw version but any similarity might just be coincidence. I do know that the number I get for fw corresponds to the lower two digits of the quoted version of the firmware(s) I've downloaded. Older fw versions had slightly different message formats and may not be correct though. I've never downgraded to test.

**Comment by martinvw** [comment]
Okay, thanks for checking it out. 

Should we maybe just prepend "10" if its a new style version? From the user it would make way more sense imho. I will also try to do an update of the rfxcom somewhere around the holidays, that will be a good test.

**Pull request merged**